tests/performance/wp-content/mu-plugins/server-timing.php

```diff
@@ -25,6 +25,8 @@ static function () use ( $server_timing_values, $template_start ) {
 
 				$server_timing_values['total'] = $server_timing_values['before-template'] + $server_timing_values['template'];
 
+				$server_timing_values['memory-usage'] = memory_get_usage();
```

Comment:
Strictly speaking, Server-Timing is not intended for this metric since it's not timing-based. But I guess it works since it's just another number? Maybe worth adding a comment to clarify regardless.

Reply:
It's a bit of a gray area, but it's totally valid and values like `Server-Timing: missedCache` or `pageType;desc="index"` (which e.g. Shopify does) are totally supported. This is nice because `Server-Timing` is only way to pass something from the server to the frontend where it can be read by JavaScript using the `PerformanceTiming` API. So it's a nice little trick :-)

Happy to add a comment if it helps 👍